### PR TITLE
Trycp tunnels admin interface

### PR DESCRIPTION
**Pull request by timotree3**
Because the holochain admin interface is only accessible to `localhost`, it is necessary for trycp to connect to it. This PR implements passing through arbitrary messages along the admin interface for trycp.

### CODE REVIEW

**Comment by timotree3** [comment]
@zippy this is ready for review. I ended up just using rust to properly encode the admin interface messages and it wasn't much work.

File: crates/trycp_server/src/admin_interface.rs [hunk 1]
```diff
@@ -0,0 +1,95 @@
+use serde::Serialize;
+use serde_derive::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::rpc_util::internal_error;
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(tag = "type")]
+enum AdminInterfaceMessage {
+    Request {
```
Comment (zippy) [inline crates/trycp_server/src/admin_interface.rs:10]: Is there really a reason to have two different types here when the content is identical?

**Review by zippy — APPROVED** [review]
This looks great.  Just one question about why two enum variants, but fine to keep.

File: crates/trycp_server/src/admin_interface.rs (hunk 1 above)
Comment (timotree3) [reply crates/trycp_server/src/admin_interface.rs:10]: I tried making a field `type: AdminInterfaceMessageType` but it would deserialize that way

**Pull request merged**